manifests/init.pp

```diff
@@ -0,0 +1,176 @@
+  Optional[Array[String[1]]]  $pkgmgr_no_vrfy                    = undef,
+  String[1]                   $pkgmgr                            = 'RPM',
+  Optional[Array[String[1]]]  $port_path_whitelist               = undef,
+  Optional[Array[String[1]]]  $port_whitelist                    = undef,
+  Optional[Array[String[1]]]  $pwdless_accounts                  = undef,
+  Optional[String[1]]         $readlink_cmd                      = undef,
+  Integer[0,1]                $rotate_mirrors                    = 1,
+  Optional[Array[String[1]]]  $rtkt_dir_whitelist                = undef,
+  Optional[Array[String[1]]]  $rtkt_file_whitelist               = undef,
+  Enum['THOROUGH','LAZY']     $scan_mode_dev                     = 'THOROUGH',
+  Optional[Enum['THOROUGH']]  $scanrootkitmode                   = undef,
+  String[1]                   $scriptdir                         = '/usr/share/rkhunter/scripts',
+  Optional[Array[String[1]]]  $scriptwhitelist                   = undef,
+  Optional[Array[String[1]]]  $shared_lib_whitelist              = undef,
+  Integer[0,1]                $show_lock_msgs                    = 1,
+  Integer[0,3]                $show_summary_time                 = 3,
+  Integer[0,1]                $show_summary_warning_number       = 0,
+  Integer[0,1]                $skip_inode_check                  = 0,
+  String[1]                   $ssh_config_dir                    = '/etc/ssh',
+  Optional[Array[String[1]]]  $startup_paths                     = undef, #suggestion: /etc/rc.d /etc/rc.local ??
+  Optional[String[1]]         $stat_cmd                          = undef,
+  Optional[Array[String[1]]]  $suspscan_dirs                     = undef,
+  Integer[0]                  $suspscan_maxsize                  = 1024000,
+  String[1]                   $suspscan_temp                     = '/dev/shm',
+  Integer[0]                  $suspscan_thresh                   = 200,
+  Optional[Array[String[1]]]  $suspscan_whitelist                = undef,
+  Optional[Array[String[1]]]  $syslog_config_file                = undef,
+  String[1]                   $tmpdir                            = '/var/lib/rkhunter',
+  Optional[Array[String[1]]]  $uid0_accounts                     = undef,
+  Array[String[1]]            $unhide_tests                      = ['sys'],
+  Array[String]             $unhidetcp_opts                    = [''],
+  Optional[Array[String[1]]]  $update_lang                       = undef,
+  Integer[0,1]                $update_mirrors                    = 1,
+  Integer[0,1]                $updt_on_os_change                 = 0,
+  Integer[0,1]                $use_locking                       = 1,
+  Optional[Array[String[1]]]  $user_fileprop_files_dirs          = undef,
+  Integer[0,1]                $use_sunsum                        = 0,
+  String[1]                   $use_syslog                        = 'NONE', #recommend authpriv.notice??
```

Comment:
`LOCAL6.NOTICE` is the SIMP default for security-related things.

`local5` is owned by something else I think.

Also, we have data types in `simplib::syslog` that should handle this appropriately.

I would strongly recommend breaking this into `syslog_facility` and `syslog_priority` and combining the appropriately in the template. This eliminates the possibility of user input errors.

Reply:
@trevor-vaughan do you mean `Simplib::Syslog::Facility` and `Simplib::Syslog::Severity` because priority seems to be the concatenation of those two already. Or should I just use `Simplib::Syslog::Priority`?

Reply:
Two separate parameters one for `Facility` and one for `Priority` that are then merged into the correct format after the fact. 

Prevents the user from making typos.

Reply:
Why both? `Facility` looks to just be the first part, and `Priority` is both parts.
Rkhunter is just looking for the full thing, no use for just the `Facility`.

Reply:
@sharkbruhaha Because they are logically different data types and programmatically combining them eliminates any possibility of user input error.

Reply:
@sharkbruhaha Apologies, I realized that I was mis-remembering how this worked. After re-review of the `simplib` types (which I should have done in the first place) your use of `Simplib;:Syslog::Priority` is correct.